engine/language_client_typescript/stream.js

```diff
@@ -57,5 +57,28 @@ class BamlStream {
         const final = await this.driveToCompletionInBg();
         return this.finalCoerce(final.parsed(false));
     }
+    /**
+     * Converts the BAML stream to a Next.js compatible stream.
+     * This is used for server-side streaming in Next.js API routes and Server Actions.
+     */
+    toStreamable() {
+        const stream = this;
+        return new ReadableStream({
```

Comment:
Where does `ReadableStream` come from? Does this break users who don't have next or react installed?

Reply:
That is built into Node, we don't need to import it from anywhere. It's available in the bowser environment and node environment. Not react specific